index.html

```diff
@@ -31,7 +31,18 @@ <h1>Welcome to the student showcase!</h1>
         <div>ELEVEN</div>
         <div>TWELVE</div>
         <div>THIRTEEN</div>
-        <div>FOURTEEN</div>
+        <div>
+            <h1>JOE ASARO</h1>
+            <img src="https://i.imgur.com/ZnCvaDN.jpg" alt="cute italian boy with trash can" width="42"><p>
+            If you want to teach people a new way of thinking, don’t bother trying to teach them. Instead, give them a tool, the use of which will lead to new ways of thinking.
+            – Richard Buckminster Fuller</p>
+                <ul>
```

Comment:
Check the indentation of this unordered list!

Reply:
what is the correct indentation?